charts/sm-operator/values.schema.json

```diff
@@ -0,0 +1,163 @@
+{
```

Comment:
Does this follow Json Schema or something similar? For whatever it is, is there a schema version we can point to?

Ex: https://json-schema.org/learn/miscellaneous-examples

Reply:
Please check now.